backend/src/services/integration-auth/integration-list.ts

```diff
@@ -73,6 +74,7 @@ export enum IntegrationUrls {
   RAILWAY_API_URL = "https://backboard.railway.app/graphql/v2",
   FLYIO_API_URL = "https://api.fly.io/graphql",
   CIRCLECI_API_URL = "https://circleci.com/api",
+  DATABRICKS_API_URL = "https:/dbc-5e0185c0-20ac.cloud.databricks.com/api",
```

Comment:
This is your personal databricks workspace url no? I had to overwrite this value with mine to get this integration to work - think we need to use their databricks workspace url from the integration form and pass it were pertinent instead of using a fixed value

Reply:
Yes, indeed. This is actually not used anymore.